doc/start/hardware-recommendations.rst

```diff
@@ -266,34 +296,46 @@ and development clusters can run successfully with modest hardware.
 +--------------+----------------+-----------------------------------------+
 |  Process     | Criteria       | Minimum Recommended                     |
 +==============+================+=========================================+
-| ``ceph-osd`` | Processor      | - 1x 64-bit AMD-64                      |
-|              |                | - 1x 32-bit ARM dual-core or better     |
+| ``ceph-osd`` | Processor      | - 1 core minimum                        |
```

Comment:
we don't test 32bit CPUs. shall we be specific on what architectures do we recommend? because we only run qa suites on AMD64. we do run "make check" on aarch64, but some tests fail now and then.

Reply:
I think we definitely want to remove the language suggesting that we support 32 bit ARM processors. ;)  I'm less sure if we should explicitly state we support AMD64 as it's pretty ubiquitous these days.  I'm a little worried that the more specific we are the more we need to keep these docs up to date and clearly we haven't been very good at it in the past. ;)

Reply:
i have been contacted recently asking if we "supported" aarch64 . so i am wondering if it'd be more convenience if i can just point them to the document.

Reply:
@markhpc: If you get an email to me with accurate information, I guarantee that I'll add it to a pull request within 12 hours of receiving the email.

Reply:
i wonder if the 'supported' question si better pointed at a doc about what hardware we build and test for